### DISCUSSION TITLE
Stealth mode + change fingerprint on demand

### DISCUSSION THREAD

**Discussion by AlJohnson1234** [kkapsner/CanvasBlocker#736] [Q&A] [answered]
I want to ensure a FP for a domain has changed when I specifically want it to, like after getting a new IP and clearing cookies. What good is doing all that if I'm not sure if the FP is going to change at the same time too? My threat model is just to hide from Big Data, so I don't mind having a persistent FP per domain as long as I can change it when I want.

Can I configure CB to do this? Is there a hack/script/setting I can do to accomplish this?

**Comment by kkapsner** [comment, marked as answer]
You can use the persistent random number generator and clear it manually:
![grafik](https://github.com/user-attachments/assets/08887b66-4b05-426f-a555-cd05896ff585)
